apps/communications/dialer/index.html

```diff
@@ -46,16 +47,24 @@
     <script defer src="/shared/js/font_size_utils.js"></script>
     <script defer src="/dialer/js/index.js"></script>
 
-    <script defer src="/shared/js/dialer/utils.js"></script>
-    <script defer src="/shared/js/dialer/tone_player.js"></script>
+    <script defer src="/shared/js/dialer/contacts.js"></script>
     <script defer src="/shared/js/dialer/dtmf_tone.js"></script>
-    <script defer src="/shared/js/dialer/keypad.js"></script>
     <script defer src="/shared/js/dialer/font_size_manager.js"></script>
+    <script defer src="/shared/js/dialer/keypad.js"></script>
+    <script defer src="/shared/js/dialer/tone_player.js"></script>
+    <script defer src="/shared/js/dialer/utils.js"></script>
+    <script defer src="/shared/js/dialer/voicemail.js"></script>
 
     <script defer src="/dialer/js/navigation.js"></script>
     <script defer src="/dialer/js/dialer.js"></script>
+    <script defer src="/dialer/js/call_log.js"></script>
```

Comment:
This was loaded by ``Navbar.ensureResources()``, see here:

https://github.com/mozilla-b2g/gaia/pull/31641/files#diff-660f9c7d029624283424917265eb27f5L46

As with the other bits these were lazy-loaded in the startup path so I just moved them to be greedy loaded instead.